File: classwork Java/NetAmount.java
import java.util.Scanner;

public class NetAmount{
	public static void main(String...args){
	Scanner input = new Scanner(System.in);

		System.out.println("PROGRAM THAT COMPUTES THE NET AMOUNT OF A BANK ACCOUNT BASED ON TRANSACTION LOG");


		String options = """

			OPTIONS\n
			1: Deposit
			2: Withdraw
			3: Get Balance

					""";

		
		int deposit = 0;
		int balance = 0;
		int condition = 1;
		int entry = 0;

		while(condition == 1){

			System.out.println(options);

			System.out.println("SELECT ONE OPTION TRO PROCEED");
			entry = input.nextInt();

			
			switch(entry){
				case 1: 
					System.out.print("Amount to be deposited: ");
					int amount = input.nextInt();
					balance += amount;
					System.out.println("You have deposited "+amount+ "and your balance is "+balance);
					break;
				case 2: 
					System.out.print("Amount to withdraw: ");
					int withdraw = input.nextInt();
					balance = balance - withdraw;

					if(balance == 0)System.out.println("Insufficient balance");
					if(withdraw > balance)System.out.println("Insufficient balance");
					if(withdraw < 0)System.out.println("Insufficient balance");
					System.out.println("You have withdrawn "+withdraw);
					break;
				case 3:
					int current_balance = balance;
					System.out.printf("Current balance = %d", current_balance);
					break;
				default:
					System.out.println("INVALID SELECTION");

			}

			System.out.println("\nDo you wish to continue? Press 1 or -1 to terminate: \n");
			condition = input.nextInt();

		}


	}

}

File: class-work/net_amount.py
print("PROGRAM THAT COMPUTES THE NET AMOUNT OF A BANK ACCOUNT BASED ON TRANSACTION LOG\n") 

name = input("Enter name to proceed: ")


options = ("""SELECT TO PROCEED: 
1: Deposit
2: Withdraw
3: Get Balance
 """)
print(options)

deposit = 0
debit = 0
balance = 0
condition = 1
entry = 0

while(condition == 1):

	entry = int(input("select option: "))

	match entry:
		case 1:
			amount = int(input("Amount to be deposited: "))
			balance += amount
			print(f"Deposit amount = {balance}")
		case 2: 
	
			debit = int(input("Amount to withdraw: "))
			balance = balance - debit

			if balance == 0:
				print("Insufficient balance")
			if debit > balance or balance == 0:
				print("Insufficient balance")
				break
			if debit < 0:
				break
			else: 
				print(f"{debit}"+" has been debited from your account")
				print(f"Your current balance = {balance}")

		case 3:
			current_balance = balance
			print(f"Current balance = {current_balance}")
		case _:
	
        		print("INVALID SELECTION")

	condition = int(input("Do you wish to continue? Press 1 or -1 to terminate: "))
	
				


 
        
			





